test-network-function/platform/suite.go

```diff
@@ -405,9 +413,40 @@ func testTainted(env *config.TestEnvironment) {
 				if err != nil {
 					message = fmt.Sprintf("Could not decode tainted kernel causes (code=%d) for node %s\n", taintedBitmap, node.Name)
 				} else {
-					message = fmt.Sprintf("Decoded tainted kernel causes (code=%d) for node %s : %s\n", taintedBitmap, node.Name, printTainted(taintedBitmap))
+					taintMsg, individualTaints := printTainted(taintedBitmap)
+
+					// We only will fail the tainted kernel check if the reason for the taint
+					// only pertains to `module was loaded`.
+					log.Debug("Checking for module was loaded taints")
```

Comment:
"Checking for module was loaded taints" => "Checking for "module was loaded" taints"

Reply:
Fixed in `ee2fa42`